.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -27,8 +27,8 @@ parameters:
     default: ""
 
   - name: LOCK_WAIT_TIMEOUT_SECONDS
-    type: string
-    default: ""
+    type: number
```

Comment:
Is it necessary to modify the data type, not sure whether it could cause calling failure

Reply:
Then let's keep using string, use number may cause unexpected issues.

Reply:
updated